build/resolver/git/entrypoint.sh

```diff
@@ -177,46 +169,37 @@ parseRevision() {
 }
 parseRevision
 
-parseCloneRes() {
-    local clone_log=$1
-    local clone_res=$2
-    cat $clone_log
-    if [ $clone_res -ne 0 ]; then
-        if grep "The requested URL returned error: 500" $clone_log > /dev/null 2>&1; then
-            # use code 99 for git clone --depth fail
-            rm -f $clone_log
-            return 99
-        else
-            exit $clone_res
-        fi
-    fi
-    return 0
+retryGitOpWithoutDepthFlagIfFromBitbucket() {
```

Comment:
Should we add some comments for why bitbucket needs to retry without depth flag?

Reply:
added.